src/makeInfinityAdder.js

```diff
@@ -6,7 +6,33 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (firstNumber) => {
+    if (firstNumber === undefined) {
```

Comment:
```suggestion
    if (!firstNumber) {
```

Reply:
But if I use !firstNumber, then the number 0 will be false, right?

Reply:
you're right, you can skip this comment